packages/calling/src/CallingClient/line/index.ts

```diff
@@ -43,13 +43,11 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
 
   public mobiusDeviceId?: string;
 
-  private mobiusUri?: string;
-
   public phoneNumber?: string;
 
   public extension?: string;
 
-  public sipAddresses?: string[];
+  public sipAddresses: string[] = [];
```

Comment:
Why was this change needed?

Reply:
As discussed, these changes is needed because this property is undefined in the web client which is giving compilation errors.